### DISCUSSION TITLE
Error during Test after fresh installation

### DISCUSSION THREAD

**Discussion by tackITEasy** [ArcadeData/arcadedb#401] [Q&A]
Hi,

I give a try for first time to ArcadeDB.

mvnw clean install

produce following:

[ERROR] Tests run: 9, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 0.015 s <<< FAILURE! - in com.arcadedb.ConstantsTest
[ERROR] com.arcadedb.ConstantsTest.getBuildNumber  Time elapsed: 0 s  <<< FAILURE!
org.opentest4j.AssertionFailedError: expected: not <null>
        at com.arcadedb.ConstantsTest.getBuildNumber(ConstantsTest.java:58)
        
Any idea what I is going wrong ?

Thanks

**Comment by lvca** [comment]
Try instead

`mvn clean install`

What's your OS and JVM version?

**Comment by arcade-player** [comment]
Did you clone the repository? 
Or did you just download the sources tar/zip?

If you start from the sources archive, rather than the clone of the repo, that test is gonna fail: our fault.
My suggestion, until we fix it, is to start from a fresh clone of the repo.

**Comment by tackITEasy** [comment]
Hello, 

Sorry for delay.

So I tried again yesterday and today with a fresh clone of the repo.
Same problem, I get an error during the tests.

java version "1.8.0_333"
Java(TM) SE Runtime Environment (build 1.8.0_333-b02)
Java HotSpot(TM) Client VM (build 25.333-b02, mixed mode)

Apache Maven 3.8.5 (3599d3414f046de2324203b78ddcf9b5e4388aa0)
Maven home: d:\dev\mvn\apache-maven-3.8.5\apache-maven-3.8.5
Java version: 18.0.1.1, vendor: Oracle Corporation, runtime: d:\dev\java\jdk\jdk-18.0.1.1
Default locale: en_US, platform encoding: UTF-8
OS name: "windows 10", version: "10.0", arch: "amd64", family: "windows"

In attachment the logs
[error.txt](https://github.com/ArcadeData/arcadedb/files/9042060/error.txt)

Thanks for your help

**Reply by lvca** [reply]
In order to build from sources, please use:

`mvn clean install -DskipTests`

Also, you need Java11 or major.
